lib/runtime-core/src/global.rs

```diff
@@ -0,0 +1,106 @@
+use crate::{
+    export::Export,
+    import::IsExport,
+    types::{GlobalDesc, Type, Value},
+    vm,
+};
+use std::{cell::RefCell, fmt, rc::Rc};
+
+pub struct Global {
+    desc: GlobalDesc,
+    storage: Rc<RefCell<vm::LocalGlobal>>,
+}
+
+impl Global {
+    pub fn new(value: Value) -> Self {
+        Self::_new(value, false)
+    }
+
+    pub fn new_mutable(value: Value) -> Self {
+        Self::_new(value, true)
+    }
+
+    fn _new(value: Value, mutable: bool) -> Self {
```

Comment:
I always think of `_` as a prefix for unused names. Is there a convention for this? If not, maybe `new_internal`